### PR TITLE
ISSUE-120: Sync 1.3.0 branch with actual 1.3.0 configs/composer/data

### PR DESCRIPTION
**Pull request by DiegoPino**
See #120

I know I am late. But better late than never

### CODE REVIEW

**Comment by alliomeria** [comment]
Awesome and thank you @DiegoPino! Will come back first thing tomorrow with a fuller review.

File: drupal/config/sync/format_strawberryfield.metadataexpose_entity.iiifmanifest.yml [hunk 1]
```diff
@@ -16,3 +16,4 @@ source_entityfield_name: null
 metadatadisplayentity_uuid: ad7aab5a-4bb3-43f2-8328-2b62895a3821
 cache: true
 active: true
+hide_on_embargo: false
```
Comment (alliomeria) [inline drupal/config/sync/format_strawberryfield.metadataexpose_entity.iiifmanifest.yml]:
```suggestion
hide_on_embargo: true
```

File: drupal/config/sync/format_strawberryfield.metadataexpose_entity.iiifmanifest3collection.yml [hunk 2]
```diff
@@ -16,3 +16,4 @@ source_entityfield_name: null
 metadatadisplayentity_uuid: 9be8473c-3fb3-42ee-aef2-431b2d4f93bb
 cache: true
 active: true
+hide_on_embargo: false
```
Comment (alliomeria) [inline drupal/config/sync/format_strawberryfield.metadataexpose_entity.iiifmanifest3collection.yml]:
```suggestion
hide_on_embargo: true
```

File: drupal/config/sync/format_strawberryfield.metadataexpose_entity.iiifmanifest3cws.yml [hunk 3]
```diff
@@ -16,3 +16,4 @@ source_entityfield_name: null
 metadatadisplayentity_uuid: 6c405928-9ae8-4557-b062-91fcf2b0af0a
 cache: true
 active: true
+hide_on_embargo: false
```
Comment (alliomeria) [inline drupal/config/sync/format_strawberryfield.metadataexpose_entity.iiifmanifest3cws.yml]:
```suggestion
hide_on_embargo: true
```

File: drupal/config/sync/format_strawberryfield.metadataexpose_entity.iiifmanifestv2.yml [hunk 4]
```diff
@@ -16,3 +16,4 @@ source_entityfield_name: null
 metadatadisplayentity_uuid: 05fe574d-7534-4261-b49b-895b94ac3fce
 cache: true
 active: true
+hide_on_embargo: false
```
Comment (alliomeria) [inline drupal/config/sync/format_strawberryfield.metadataexpose_entity.iiifmanifestv2.yml]:
```suggestion
hide_on_embargo: true
```

**Comment by DiegoPino** [comment]
@alliomeria thanks so much for the code suggestions. This all makes sense. Just merged your commits!

**Comment by alliomeria** [comment]
Thanks to you @DiegoPino!

**Pull request merged**